public/script.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+let secretNumber;
+let currentScore = 20;
+let highScore = 0;
+const initialMessage = 'Start guessing...';
+
+function setTextContent(targetClass, newContent) {
+  return document.querySelector(`.${targetClass}`).textContent = newContent;
+}
+function generateNewSecretNumber() {
+  secretNumber = Math.trunc(Math.random() * 20) + 1;
+}
+
+generateNewSecretNumber();
+
+document.querySelector('.btn__check').addEventListener('click', function () {
+  const guess = Number(document.querySelector('.guess-input').value);
+
+  // If player guesses correctly (wins) ---------------------------------------
+  if (guess === secretNumber) {
+    setTextContent('secret-number', secretNumber);
+    setTextContent('message', 'You guessed it! 🎉');
+    document.querySelector('body').style.background = '#60b347';
+    document.querySelector('.secret-number').style.width = '30rem';
+    if (currentScore > highScore) {
+      highScore = currentScore;
+      setTextContent('highscore', highScore);
+    }
+
+  // If guess is invalid ---------------------------------------
+    // When there is no input
+  } else if (!guess) {
+    setTextContent('message', 'Please type a number into the box on the left');
+    // When guess is not between 1 - 20
+  } else if (guess > 20 || guess < 1) {
```

Comment:
```suggestion
  } else if (guess < 1 || guess > 20) {
```
This is closer to natural language: "If guess is less than 1 or greater than 20" rather than "if guess is greater than 20 or less than 1".

Reply:
Feedback adopted!